README.md

```diff
@@ -196,6 +196,7 @@ _Related: [Internet of Things (IoT)](#internet-of-things-iot), [Software Develop
 - [homebank-converter](https://github.com/Binnette/homebank-converter) - Web app to convert an export bank file to compatible Homebank csv. ([Demo](https://binnette.github.io/homebank-converter/)) `AGPL-3.0` `HTML5`
 - [HRConvert2](https://github.com/zelon88/HRConvert2) - Drag-and-drop file conversion server with session based authentication, automatic temporary file maintenance, and logging capability. `GPL-3.0` `PHP`
 - [Huginn](https://github.com/huginn/huginn) - Allows you to build agents that monitor and act on your behalf. `MIT` `Ruby`
+- [ifconfig.io](https://ifconfig.io/) - Information on your connection. ([Source Code](https://github.com/georgyo/ifconfig.io)) `MIT` `Go/Docker`
```

Comment:
> https://github.com/georgyo/ifconfig.io#deployment

Deployment only lists Docker guide.

The description should explain what the software does. I have added the description of the repo and cleaned it up a little. 

```suggestion
- [ifconfig.io](https://ifconfig.io/) - Provides information on your connection. Inspired by ifconfig.me, but designed for pure speed. A single server can do 18,000 requests per seconds while only consuming 50 MBs of RAM. ([Source Code](https://github.com/georgyo/ifconfig.io)) `MIT` `Docker`
```